index.html

```diff
@@ -340,6 +340,139 @@ <h4 class="functionality__paragraph">Necesito Ayuda</h4>
         </div>
       </div>
     </section>
+    <section class="most-popular">
+      <h3 class="most-popular__title">Lo más popular</h3>
+      <div class="most-popular__products">
+        <div class="most-popular__left">
+          <div class="most-popular__card">
+            <a href="#" class="most-popular__link"></a>
+            <button class="most-popular__button"> SOLO WEB</button>
```

Comment:
Esto es solo informativo, no necesitas poner un boton. Es mas, yo lo llamaria `badge` en vez de `button`
```suggestion
            <p class="most-popular__badge">SOLO WEB</p>
```